express/index.cjs

```diff
@@ -447,7 +433,7 @@ app.put('/reset-password', async (req, res) => {
     }
```

Comment:
How do I properly query the id from either table and define that id onto the jwt id?

Reply:
I created the checkUser helper function to not only check if the user exists, but also return the user_id if the check finds the user.